src/app/pages/wallets/wallet-details/pages-wallet-details.service.ts

```diff
@@ -20,4 +20,16 @@ export class PagesWalletDetailsService {
       )),
     );
   }
+
+  public createWalletTransaction(data: ITransactionPayload): Observable<WalletsDetailsTransaction> {
+    return this.gateway.createWalletTransaction(data).pipe(
+      map(transactionsResp => new WalletsDetailsTransaction(transactionsResp)),
+    );
+  }
+
+  public editWalletTransaction(id: number, data: WalletsDetailsTransaction): Observable<WalletsDetailsTransaction> {
```

Comment:
Id instead `number` should be `TServerEntityId`. And this should be actually changed also for models, wallets and other places where `number` is used instead `TServerEntityId` - if backend will change from numeric IDs to string based (like UUID) then we will have to change this only in one place.